backend/main.py

```
@@ -1,369 +1,305 @@
-        # raise # Uncomment to make Message Bus connection critical for startup
 
-    # Start agent listeners that depend on message bus
     agent_listeners_started = True
     try:
         logger.info("Starting DispatcherAgent listeners...")
         await dispatcher_agent.start_listening()
-        logger.info("DispatcherAgent listeners started.")
     except Exception as e:
-        logger.error(f"Failed to start DispatcherAgent listeners: {e}", exc_info=True)
-        agent_listeners_started = False # Mark as failed
+        logger.error(f"Failed to start DispatcherAgent listeners: {e}", exc_info=True); agent_listeners_started = False
```

Flake8 (7.2.0)

[error] multiple statements on one line (semicolon)

(E702)